packages/color-area/README.md

```diff
@@ -0,0 +1,67 @@
+## Description
+
+An `<sp-color-area>` allows users to visually select two properties of a color simultaneously. It's commonly used together with a color slider or color wheel.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-area?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-area)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-area?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-area)
+
+```
+yarn add @spectrum-web-components/color-area
+```
+
+Import the side effectful registration of `<sp-color-area>` via:
+
+```
+import '@spectrum-web-components/color-area/sp-color-area.js';
+```
+
+When looking to leverage the `ColorArea` base class as a type and/or for extension purposes, do so via:
+
+```
+import { ColorArea } from '@spectrum-web-components/color-area';
+```
+
+## Color Formatting
+
+When using the color elements, use `el.color` to access the `color` property, which should manage itself in the colour format supplied. For example, If you supply a color in `rgb()` format, `el.color` should return the color in `rgb()` format, as well. In ColorArea, colours are formatted as hex values.
+
+The current color formats supported are as follows:
+
+-   Hex3, Hex4, Hex6, Hex8
+-   HSV, HSVA
+-   HSL, HSLA
+-   RGB, RGBA
+-   Strings (eg "red", "blue")
+
+**Please note for the following formats: HSV, HSVA, HSL, HSLA**
+When setting a color's lightness or value to 100%, the hue and saturation value are not preserved. This is detailed in the [TinyColor documentation](https://www.npmjs.com/package/@ctrl/tinycolor). Currently, the Spectrum Web Components has a workaround to support the preservation of the hue.
```

Comment:
Is it `lightness` and `value` at 100% or is it one at 100% and the other at 0%?

Can we say `may not be preserved` instead, and connect it directly to the `color` and `value` properties of the element?

`color-area` does not manage `hue` directly, can you remove this last sentence.